.github/workflows/ci_integration.yml

```diff
@@ -294,6 +295,8 @@ jobs:
     steps:
       - name: "Check out repository code"
         uses: "actions/checkout@v2"
+      - name: "Check & Build Dependencies Container Image"
+        uses: "./.github/workflows/build_dependency_image_conditional.yml"
```

Comment:
Is this path right? Relative to the repository base directory, rather than to the directory the current workflow is defined in?

Reply:
Yes, I've tested this here: https://github.com/nautobot/nautobot-plugin-netutils/pull/2/files#diff-d61e1018c78f1c276c1f4ca893c840c6708bcbe7a364c359540eb764802d5e35R12